tests/TestCase.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Ratchet\RFC6455\Test;
+
+class TestCase extends \PHPUnit_Framework_TestCase
```

Comment:
There is no purpose right now. When I have setup testing on other projects, I sometimes end up using it to put common test stuff in there. I can remove it - just let me know.